client/js/global.js

```diff
@@ -73,18 +77,18 @@ function fmOverlay() {
   if ($overlay.length) {
     setTimeout(() => {
       /**
-       * Check & Handle Image Overlays
+       * Display Overlay
+       * adds class to make overlay active then checks
+       * if overlay is an image type
        */
-      if ($overlay.hasClass('fm-overlay-image')) {
-        // handle image overlay resizing
+      if ($overlay.show().addClass(activeClass).hasClass('fm-overlay-image')) {
```

Comment:
Sequentially chains together the display of the modal to the resizing of the contained image.  This should prevent against race conditions where `resizeOerlayImage()` is fired before overlay is visible.